### ISSUE TITLE
HTML5?

### ISSUE THREAD

**Issue by subsoap** [defold/extension-firebase-analytics#17]
Description says only iOS/Android support. Is HTML5 / desktop platforms possible to support?

https://firebase.google.com/docs/web/setup

**Comment by britzl** [comment]
Yep, it's possible to also support HTML5 through the JavaScript SDK, in a similar way to how we support HTML5 for the Facebook extension.
